### DISCUSSION TITLE
that scope name is already in use

### DISCUSSION THREAD

**Discussion by simpleneeraj** [npm/feedback#678] [Registry] [answered]
![Screenshot_20220509-085339_Chrome](https://user-images.githubusercontent.com/69778720/167335400-cdccb5ea-15cb-4dd8-b7ac-7efa59b36e64.png)

### I faced this problem 
I deleted all my old accounts from npm but when I created another account with my old username I get this error.

`Shown in image`

But I search for this scope name it doesn't exist in npm registry or database. 

I want solutions as soon as possible 😩

**Comment by monishcm** [comment, marked as answer]
Please try contacting the support team.

**Comment by StrangeNoob** [comment]
@simpleneeraj I am facing the same problem. Can you help me with this ?

**Reply by MylesBorins** [reply]
Please contact support at https://npmjs.com/support

**Comment by l-Zetta-l** [comment]
@monishcm @MylesBorins Our team facing a serious problem and urgently waiting for help. Are there all the tickets still processing actually? How long do we need to wait?
